Corona-Warn-App/src/test/java/de/rki/coronawarnapp/submission/TransmissionRiskVectorDeterminatorTest.kt

```diff
@@ -16,13 +16,4 @@ class TransmissionRiskVectorDeterminatorTest {
             ).raw
         )
     }
-
```

Comment:
Ask: while I see why `test_numberOfDays` was removed, this class definitely needs more test cases, e.g. these ones here: https://github.com/corona-warn-app/cwa-app-ios/blob/8bb7273853b598a507c89acd45dad0e4157e73e7/src/xcode/ENA/ENA/Source/Models/__tests__/SymptomsOnsetTests.swift#L29-L195